bundles/org.openhab.binding.gree/src/main/java/org/openhab/binding/gree/internal/GreeBindingConstants.java

```diff
@@ -174,4 +174,12 @@ public class GreeBindingConstants {
      *      for more details.
      */
     public static final double INTERNAL_TEMP_SENSOR_OFFSET = -40.0;
+
+    public static enum ENCRYPTION_TYPES {
```

Comment:
Not much familiar with Openhab naming convention but in typical Java convention for enum it should be `EncryptionType`

Reply:
openHAB requires static members to be in capitals. But i don't think it has any benefit marking it as static, so indeed you might want to change it.
```suggestion
    public enum EncryptionType {
```
For reference: https://stackoverflow.com/a/23128025

Reply:
It should be static but this is not a member it is definition of an Enum. Does it not pass some validation? You can also extract it as its own separate `.java` file.

Reply:
I used capital letters because of the other static members in the Constant file ( I was really wondering if it should be with capital or `EncryptionType`), I did run all the validations and build and everything passed. It looks too small for a separate file and I haven't seen anywhere else used a separate file for enum.